### DISCUSSION TITLE
Zeek CaptureLoss::watch_interval error

### DISCUSSION THREAD

**Discussion by kiaraho** [Security-Onion-Solutions/securityonion#4923] [Unsupported Versions] [answered]
Zeek is failing after upgrading to `2.3.61`. The `docker logs so-zeek` command shows the following output:
```
checking configurations ...
logger scripts failed.
error in /opt/zeek/share/zeek/site/local.zeek, line 4: "redef" used but not previously defined (CaptureLoss::watch_interval)

manager scripts failed.
error in /opt/zeek/share/zeek/site/local.zeek, line 4: "redef" used but not previously defined (CaptureLoss::watch_interval)

proxy scripts failed.
error in /opt/zeek/share/zeek/site/local.zeek, line 4: "redef" used but not previously defined (CaptureLoss::watch_interval)

worker-1-1 scripts failed.
error in /opt/zeek/share/zeek/site/local.zeek, line 4: "redef" used but not previously defined (CaptureLoss::watch_interval)
```
Contents of the `/opt/zeek/share/zeek/site/local.zeek` file shown below:
```
##! Local site policy.
@load protocols/conn/mac-logging
@load-sigs frameworks/signatures/detect-windows-shells
redef CaptureLoss::watch_interval = 5 mins;
redef LogAscii::use_json = T;
```
Other than enabling mac-logging via `/opt/so/saltstack/local/pillar/global.sls` on the Manager, no other changes have been made to the sensor.

**Comment by defensivedepth** [comment, marked as answer]
I would suggest backing out your custom config and restarting Zeek to see if the error goes away.

**Reply by kiaraho** [reply]
Reverting to the default Zeek config resolved the issue. Did anything change in `2.3.61` that affects the enabling of MAC address logging via the `@load protocols/conn/mac-logging` line? We have had this custom config since `2.3.30` with no issues.

We enabled this custom config by adding the following lines to the end of `/opt/so/saltstack/local/pillar/global.sls` on the Manager node.
```
zeek:
  local:
    '@load':
      - protocols/conn/mac-logging
```

**Reply by defensivedepth** [reply]
You need to add the entire default @load config into the global.sls file, and then add your custom one at the bottom, then restart Zeek.  See here: https://docs.securityonion.net/en/2.3/zeek.html#custom-scripts

That should take care of that redef error; I am unsure why it is just showing up now.

**Reply by kiaraho** [reply]
Thanks!
